source/index.html.md

```diff
@@ -1301,6 +1343,79 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
     <div class="type">May not be all digits, all same character, or all sequential characters (e.g. "abc")</div>
     <div class="type">Optional</div>
   </dd>
+  <dt>travel
+    <span class="type">[\x20-\x7E]{2,22}
+      <a target="_blank" href="https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Text about travel will go here
+  </dd>
+</dl>
+
+##### Travel addendum data
+
+For service type Lodging the following fields are relevant
+
+<dl class="dl-vertical">
+  <dt>
+    travel[lodging][no_show]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    An indicator that the cardholder is being billed for a reserved room that was not actually
+    rented (AKA a "no-show" charge). The issuer may provide this information on the customer
+    statement or use it to enhance customer service information.
+  </dd>
+  <dt>
+    travel[lodging][extra_charge]
+    <span class="type">(<code>no_extra_charge</code>|<code>restaurant</code>|<code>gift_shop</code>|<code>mini_bar</code>|<code>telephone</code>|<code>other</code>|<code>laundry</code>)</span>
+  </dt>
+  <dd>
+    This field may be used to indicate the type of additional charges added to a customer's bill
+    after check-out. Each position in the field can be used to indicate a type of charge. If there
+    are less than six additional charges, they must be left-justified, space-filled or zero-filled
+    to the right.
+  </dd>
+  <dt>
+    travel[lodging][check_in]
+    <span class="type">[0-9]{6}</span>
+  </dt>
+  <dd>
+    The date the customer checked into the hotel, or in the case of a no-show or an advance lodging,
+    the scheduled arrival date. The entry must be a six-digit numeric, either zeros or a valid date,
+    in the format YYMMDD.
+  </dd>
```

Comment:
Elsewhere, we use 4 digit year; what about `YYYY-MM-DD` (which is slightly more "[ISO8601](https://en.wikipedia.org/wiki/ISO_8601)-ish")? One should want the timezone also, but I guess the world ignores that fact.

Reply:
Yea, any date would do. But maybe TAD data is stored like `YYMMDD` by the PSP's already because of other acquirers? But conversion i trivial, so we should maybe enforce `YYYY-MM-DD` for consistency.